lightly/transforms/simclr_transform.py

```diff
@@ -135,7 +129,7 @@ def __init__(
             transform += [T.Normalize(mean=normalize["mean"], std=normalize["std"])]
         self.transform = T.Compose(transform)
 
-    def __call__(self, image: Tensor) -> Tensor:
+    def __call__(self, image: Union[Tensor, Image]) -> Tensor:
```

Comment:
Can the output not be an image as well?

Reply:
No, it can't. Otherwise image normalisation gives error. The ToTensor is optional only if the input is already a tensor

Reply:
Image normalization is also optional though.

Reply:
That's also true, but I wouldn't let this possibility as it seems a bit tacky to me. Also in the collate we assume that the batches are of tensors.